packages/data-context/src/data/coreDataShape.ts

```diff
@@ -219,5 +224,25 @@ export function makeCoreData (modeOptions: Partial<AllModeOptions> = {}): CoreDa
     packageManager: 'npm',
     forceReconfigureProject: null,
     versionData: null,
+    cloud: {
+      testsForRunResults: [
+        {
+          'titlePath': 't1',
+          'status': 'FAILED',
+        },
+        {
+          'titlePath': 't2',
+          'status': 'FAILED',
+        },
+        {
+          'titlePath': 's1 t3',
+          'status': 'FAILED',
+        },
+        {
+          'titlePath': 's1 t4',
+          'status': 'FAILED',
+        },
+      ],
+    },
```

Comment:
This feels like test data - is this meant to be here?

Reply:
I've added a comment but this is just some test data as it was never hooked up to the cloud.